main.js

```diff
@@ -435,6 +483,99 @@ class ModuleInstance extends InstanceBase {
 
 		this.sendOsc(`${prefix}/${id}${suffix}`, arg)
 	}
+	
+	getDistinceParamForWheelLabel( wheel_label ) {
+		// Below are regex tests for the label name in a switch instead of
+		// cascading if statements. These are case insensitive tests
+		let distinctparam = ''
+		switch ( true ) {
```

Comment:
doing this as a switch rather than an if statement is a bit odd, but im not going to try and change it.
you could also return early instead of setting `distinctparam` and returning it at the end.
Im also not keen on the use of regex here, as it feels a bit unnecessary, but Im not going to say it has to be changed

Reply:
Understood, when I started this process its seemed that there variations that needed to be covered and in the end, at least for this set, they turned out to be fixed. After all that, I agree it could be refactored and perhaps I can put that on my list for the future.